app/views/users/_unfollow.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= form_for(current_user.actvive_relationships.find_by(followed_id: @user.id),
```

Comment:
OF COURSE there will be an error.
There is wrong spelling.
Double-check your spelling.